src/renderer/renderScene.ts

```diff
@@ -163,6 +163,7 @@ export function renderScene(
   // Paint remote pointers
   for (const clientId in sceneState.remotePointerViewportCoords) {
     let { x, y } = sceneState.remotePointerViewportCoords[clientId];
+    const username = sceneState.remotePointerUsernames[clientId];
```

Comment:
do you think it'd be better to store the username along with the x and y?

Reply:
I honestly don't know...

Reply:
I was thinking about what else we could store here, some ideas:
- mouse up/down status
- current cursor